tests/foreman/cli/test_puppetmodule.py

```diff
@@ -1,117 +0,0 @@
-"""Test class for PuppetModule CLI
-
-:Requirement: Puppetmodule
-
-:CaseAutomation: Automated
-
-:CaseLevel: Acceptance
-
-:CaseComponent: Puppet
-
-:Assignee: vsedmik
-
-:TestType: Functional
-
-:CaseImportance: High
-
-:Upstream: No
-"""
-import pytest
-
-from robottelo.cli.factory import make_product
-from robottelo.cli.factory import make_repository
-from robottelo.cli.puppetmodule import PuppetModule
-from robottelo.cli.repository import Repository
-from robottelo.config import settings
-from robottelo.constants.repos import FAKE_0_PUPPET_REPO
-from robottelo.constants.repos import FAKE_1_PUPPET_REPO
```

Comment:
One would imagine we won't need these constants anymore but we need to check other occurrences